cli/determined_cli/experiment.py

```diff
@@ -280,7 +280,7 @@ def describe(args: Namespace) -> None:
         v_metrics_headers = []
 
     headers = (
-        ["Trial ID", "Step ID", "State", "Start Time", "End Time"]
+        ["Trial ID", "Train Workload #", "State", "Start Time", "End Time"]
```

Comment:
The webui calculates the number of batches that have been completed in some places, rather than showing the Step ID.  It seems like `Train Workload #` is a 1:1 replacement for `Step ID`, but it might not any more useful to users than if we just left it as `Step ID`.

How hard would it be to replace these headers with calculations of the number of batches at each point?

Reply:
Done. It's a bit of work but it's much more useful. The new cli may be a few release away and there's no reason to leave it in a bad state for that long.